src/EditorFeatures/CSharpTest/Wrapping/ParameterWrappingTests.cs

```diff
@@ -984,6 +984,42 @@ public async Task TestWithMissingParameterList()
             await TestMissingAsync(
 @"class C {
     public void UpsertRecord<T>[||]
+}");
+        }
+
+        [Fact, WorkItem(61362, "https://github.com/dotnet/roslyn/issues/61362")]
```

Comment:
This ID is different than the bug listed in the summary

Reply:
whoops.  will fix.